Server 1/smartface-master/sf-docker/docker-compose.yml

```diff
@@ -785,86 +786,14 @@ services:
     volumes:
       - "./iengine.lic:/etc/innovatrics/iengine.lic"
 
-  detector-cpu-1:
-    image: ${REGISTRY}sf-detector:${SF_VERSION}
-    container_name: SFDetectCpu1
-    restart: unless-stopped
-    environment:
-      - RabbitMQ__Hostname
-      - RabbitMQ__Username
-      - RabbitMQ__Password
-      - RabbitMQ__Port
-      - AppSettings__Log-RollingFile-Enabled=false
-      - AppSettings__USE_JAEGER_APP_SETTINGS
-      - JAEGER_AGENT_HOST
-      # - Gpu__GpuEnabled=true
-      # - Gpu__GpuNeuralRuntime=Tensor
-    volumes:
-      - "./iengine.lic:/etc/innovatrics/iengine.lic"
-    # - "/var/tmp/innovatrics/tensor-rt:/var/tmp/innovatrics/tensor-rt"
-    #runtime: nvidia
-
-  detector-gpu-1:
-    image: ${REGISTRY}sf-detector:${SF_VERSION}
-    container_name: SFDetectGpu1
-    restart: unless-stopped
-    environment:
-      - RabbitMQ__Hostname
-      - RabbitMQ__Username
-      - RabbitMQ__Password
-      - RabbitMQ__Port
-      - AppSettings__Log-RollingFile-Enabled=false
-      - AppSettings__USE_JAEGER_APP_SETTINGS
-      - JAEGER_AGENT_HOST
-      - Gpu__GpuEnabled=true
-      # - Gpu__GpuNeuralRuntime=Tensor
-    volumes:
-      - "./iengine.lic:/etc/innovatrics/iengine.lic"
-    # - "/var/tmp/innovatrics/tensor-rt:/var/tmp/innovatrics/tensor-rt"
-    runtime: nvidia
-
-  detector-cpu-2:
-    image: ${REGISTRY}sf-detector:${SF_VERSION}
-    container_name: SFDetectCpu2
-    restart: unless-stopped
-    environment:
-      - RabbitMQ__Hostname
-      - RabbitMQ__Username
-      - RabbitMQ__Password
-      - RabbitMQ__Port
-      - AppSettings__Log-RollingFile-Enabled=false
-      - AppSettings__USE_JAEGER_APP_SETTINGS
-      - JAEGER_AGENT_HOST
-      # - Gpu__GpuEnabled=true
-      # - Gpu__GpuNeuralRuntime=Tensor
-    volumes:
-      - "./iengine.lic:/etc/innovatrics/iengine.lic"
-    # - "/var/tmp/innovatrics/tensor-rt:/var/tmp/innovatrics/tensor-rt"
-    #runtime: nvidia
-
-  detector-gpu-2:
-    image: ${REGISTRY}sf-detector:${SF_VERSION}
-    container_name: SFDetectGpu2
-    restart: unless-stopped
-    environment:
-      - RabbitMQ__Hostname
-      - RabbitMQ__Username
-      - RabbitMQ__Password
-      - RabbitMQ__Port
-      - AppSettings__Log-RollingFile-Enabled=false
-      - AppSettings__USE_JAEGER_APP_SETTINGS
-      - JAEGER_AGENT_HOST
-      - Gpu__GpuEnabled=true
-      # - Gpu__GpuNeuralRuntime=Tensor
-    volumes:
-      - "./iengine.lic:/etc/innovatrics/iengine.lic"
-    # - "/var/tmp/innovatrics/tensor-rt:/var/tmp/innovatrics/tensor-rt"
-    runtime: nvidia
-
-  detector-cpu-3:
+  # INNO: all services except camera can be scaled with scale parameter
+  detector-cpu:
     image: ${REGISTRY}sf-detector:${SF_VERSION}
-    container_name: SFDetectCpu3
+    # if you want spawn more detectors, container_name must be removed. It is not required.
+    # container_name: SFDetectCpu
     restart: unless-stopped
+    # INNO: uncomment if you want to scale more detectors
+    # scale: 3
```

Comment:
!!!! CAUTION !!!!
The scale number is just for demonstrating purposes. You have to replace scale: 3 with number of services calculated by sizing tool.